subnet/deployment-generator/src/faucet.js

```diff
@@ -0,0 +1,175 @@
+process.chdir(__dirname);
+const fs = require("fs");
+const ethers = require("ethers");
+const path = require("path");
+const { error } = require("console");
+const express = require("express");
+const app = express();
+const router = express.Router();
+
+const usage =
+  "Usage: SUBNET_URL=<subnet_url> node faucet.js <from wallet pk> <to wallet pubkey> <token amount>";
+const subnetURL = process.env.SUBNET_URL || "";
+if (subnetURL == "") {
+  console.log("Invalid SUBNET_URL");
+  console.log(usage);
+  process.exit();
+}
+console.log("RPC url:", subnetURL);
+
+const inputs = process.argv;
+// console.log("inputs", inputs)
+
+if (inputs.length == 5) {
+  try {
+    processTransfer(inputs).then(output => {
+      // console.log(output)
+      process.exit()
+    })
+  } catch (error) {
+    console.log(error);
+    console.log(usage);
+    process.exit();
+  }
+} else if (inputs.length == 4) {
+  faucetServer(inputs);
```

Comment:
Why use input length to define different usage? better to use --flag

Reply:
right, this is rush job, my bad
Is it ok to change in the next update? Other components also need updating, I'm afraid to cause bugs right now.
This part is not seen/used by the user.